README.md

```diff
@@ -163,6 +163,19 @@ The `commentstart` linter can automatically fix comments that do not start with
 
 When the `json` tag is present, and matches the first word of the field comment in all but casing, the linter will suggest that the comment be updated to match the `json` tag.
 
+## DuplicateMarkers
+
+The duplicatemarkers linter checks for exact duplicates of markers for types and fields.
+This means that something like:
+
+// +kubebuilder:validation:MaxLength=10
+// +kubebuilder:validation:MaxLength=10 
+Will be flagged by this linter, while something like:
+
+// +kubebuilder:validation:MaxLength=10
+// +kubebuilder:validation:MaxLength=11
+will not.
+
```

Comment:
Include a `### Fixes` section for so we document what suggested fixes occur as part of this linter